vars/virtualenv.groovy

```diff
@@ -1,6 +1,16 @@
 import com.ableton.VirtualEnv
 
 
+/**
+ * Create a new {@link com.ableton.VirtualEnv} instance.
+ * @param python Python version or absolute path to Python executable.
+ * @param pyenvRoot Path of the pyenv installation.
+ * @return New {@link com.ableton.VirtualEnv} instance.
+ */
+VirtualEnv create(String python, String pyenvRoot) {
```

Comment:
I think it would be nicer to have a `createWithPyenv` function that would only take the `python` version.

Reply:
Fixed in e29dd17f52ce9b759b342e37fcdab1aa79e9e62a (though as we discussed on the VC, not as you suggested above).